app/graph/types/source_type.rb

```diff
@@ -1,47 +1,115 @@
-SourceType = GraphqlCrudOperations.define_default_type do
-  name 'Source'
-  description 'Source type'
-
-  interfaces [NodeIdentification.interface]
-
-  field :image, types.String
-  field :description, !types.String
-  field :name, !types.String
-  field :dbid, types.Int
-  field :user_id, types.Int
-  field :permissions, types.String
-  field :pusher_channel, types.String
-  field :lock_version, types.Int
-  field :medias_count, types.Int
-  field :accounts_count, types.Int
-  field :overridden, JsonStringType
-  field :archived, types.Int
-
-  connection :accounts, -> { AccountType.connection_type } do
-    resolve ->(source, _args, _ctx) { source.accounts }
-  end
+# require "inclusions/task_and_annotation_fields"
+
+class SourceType < DefaultObject
+  # include TaskAndAnnotationFields
```

Comment:
TODO: Figure out what's going on here and clean up - should be covered by TaskAndAnnotationFields